### PR TITLE
common: drop/mark-as-final getters of buffer::raw for palign.

### PR DESCRIPTION
**Pull request by rzarzynski**

### CODE REVIEW

File: src/include/buffer_raw.h [hunk 1]
```diff
@@ -92,12 +92,9 @@ namespace ceph::buffer {
     virtual int zero_copy_to_fd(int fd, loff_t *offset) {
       return -ENOTSUP;
     }
-    virtual bool is_page_aligned() {
+    virtual bool is_page_aligned() final {
```
Comment (rzarzynski) [inline src/include/buffer_raw.h]: Marked as `final` solely for testing. Next commit will strip `virtual`-ness from this method.

**Comment by rzarzynski** [comment]
I propose to merge this one after #24031 due to potential conflict.

**Comment by tchaikov** [comment]
- http://pulpito.ceph.com/kchai-2018-09-14_03:44:53-rados-wip-kefu-testing-2018-09-14-0938-distro-basic-mira/
- http://pulpito.ceph.com/kchai-2018-09-15_11:45:38-rados-wip-kefu-testing-2018-09-14-0938-distro-basic-mira/
- http://pulpito.ceph.com/kchai-2018-09-16_00:58:57-rados-wip-kefu-testing-2018-09-14-0938-distro-basic-mira/
- http://pulpito.ceph.com/kchai-2018-09-16_11:53:24-rados-wip-kefu-testing-2018-09-14-0938-distro-basic-mira/
- http://pulpito.ceph.com/kchai-2018-09-16_14:12:08-rados-wip-kefu-testing-2018-09-14-0938-distro-basic-mira/
- http://pulpito.ceph.com/kchai-2018-09-17_04:41:34-rados-wip-kefu-testing-2018-09-16-2359-distro-basic-mira/
- http://pulpito.ceph.com/kchai-2018-09-17_07:23:58-rados-wip-kefu-testing-2018-09-16-2359-distro-basic-smithi/

the failed tests were

- caused by #19454 or
- fixed by https://github.com/ceph/teuthology/pull/1211  or
- fixed by https://github.com/ceph/ceph/pull/24057

**Comment by tchaikov** [comment]
@rzarzynski i am sorry, i merged this PR by accident. i rebased your #24031 onto master at https://github.com/tchaikov/ceph/tree/wip-common-bl-drop-0c . hopefully this could make your rebase easier.

**Comment by rzarzynski** [comment]
@tchaikov: no worries and thanks for the branch! My apologies for not setting the `DNM` on this one.

**Pull request merged**